python/doc/examples/data_analysis/sample_analysis/plot_multicollinearity.py

```
@@ -0,0 +1,166 @@
+
+import openturns as ot
+import openturns.viewer as otv
+from openturns.usecases import ames_housing
+import matplotlib
+
+# %%
+# We will also perform bootstraps and plot the results, so we start by defining some helper functions.
+# First, we define one that computes the desired quantity:
+
+
+def compute_quantity(X, Y, type):
```

Ruff (0.16.1)

[error] Function argument `type` is shadowing a Python builtin

(A002)